browser/src/dom/NotebookbarAccessibilityDefinitions.js

```diff
@@ -0,0 +1,220 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+	This class is used as the notebookbar accessibility configuration provider for the current document type.
+*/
+
+/* global app */
+
+/* eslint-disable-next-line */
+var NotebookbarAccessibilityDefinitions = function() {
+
+	this.getWriterDefinitions = function() {
+		return {
+			'File-tab-label': {
+				focusBack: true,
+				combination: '0',
+				contentList: []
+			},
+			'Home-tab-label': {
+				focusBack: true,
+				combination: '1',
+				contentList: [
+					{ id: 'ResetAttributesimg',				focusBack: true, 	combination: 'AA' },
```

Comment:
one more improvement I can see is to avoid adding "img" here everywhere but instead use some smart selector in querySelector, but that's for the future cleanup I think